packages/web/components/InlineFeedbackPopover/Comment.tsx

```diff
@@ -281,6 +324,12 @@ const Comment: React.FC<CommentProps> = ({ comment, canEdit, onUpdateComment, cu
         .like-btn :global(svg:hover) {
           cursor: pointer;
         }
+        .progress {
+          cursor: progress;
+        }
+        .pointer-events-none {
+          pointer-events: none;
+        }
```

Comment:
This is kind of nitpicky, but you can save yourself the extra class with something like:

```suggestion
        .progress > .like-btn {
          pointer-events: none;
        }
```

Reply:
that's a good one! 💪🏼